python/tests/test_logistic_regression.py

```diff
@@ -1,10 +1,20 @@
+import math
+import os
+import sys
+
+file_path = os.path.abspath(__file__)
```

Comment:
Would it be better/cleaner to set PYTHONPATH env var in run_test.sh .

Reply:
Yeah, revised to use PYTHONPATH env variable.